integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -36,11 +37,11 @@
   private static StringBuffer namespaceList;
 
   /**
-   * This method gets called only once before any of the test methods are executed. It does the
-   * initialization of the integration test properties defined in OperatorIT.properties and setting
-   * the resultRoot, pvRoot and projectRoot attributes.
+   * This method gets called only once before any of the test methods are executed
+   * It does the initialization of the integration test properties defined in 
+   * OperatorIT.properties and define resultRoot,pvRoot and projectRoot attributes.
```

Comment:
comma after space

Reply:
OperatorIT.properties and define resultRoot, pvRoot, and projectRoot attributes.